// Copyright 2015, Google Inc.
// All rights reserved.


syntax = "proto3";

// interface de servi√ßo
service Agenda {
  rpc add_contact (Contact) returns (Reply) {}
  rpc delete_contact (Contact) returns (Reply) {}
  rpc edit_contact (Contact) returns (Reply) {}
  rpc list_contacts (EmptyRequest) returns (stream Contact) {}
}

message Contact {
	string name = 1;
	string phone_number = 2;
}

message EmptyRequest {}

message Reply {
  string message = 1;
}
